src/views/Activities.vue

```diff
@@ -0,0 +1,214 @@
+            v-for="index in Math.ceil(coursesData.length / 3) + 1"
+            :key="index"
+          >
+            <b-col sm="12" md="4" v-for="id in 3" :key="id">
+              <Activity
+                class="activity"
+                v-if="getTranslatedId(id, index) < coursesData.length"
+                :title="coursesData[getTranslatedId(id, index)].title"
+                :description="
+                  coursesData[getTranslatedId(id, index)].description
+                "
+                :date="coursesData[getTranslatedId(id, index)].date"
+              />
+            </b-col>
+          </b-row>
+        </div>
+      </b-container>
+    </section>
+    <section>
+      <b-container class="activity-section">
+        <h2 class="activity-section-heading" data-aos="fade-down">Workshops</h2>
+        <div class="activity-cards" data-aos="flip-right" data-aos-duration="800">
+          <b-row
+            v-for="index in Math.ceil(workshopData.length / 3) + 1"
+            :key="index"
+          >
+            <b-col sm="12" md="4" v-for="id in 3" :key="id">
+              <Activity
+                class="activity"
+                v-if="getTranslatedId(id, index) < workshopData.length"
+                :title="workshopData[getTranslatedId(id, index)].title"
+                :description="
+                  workshopData[getTranslatedId(id, index)].description
+                "
+                :date="workshopData[getTranslatedId(id, index)].date"
+                :backgroundColor="'#eee'"
+              />
+            </b-col>
+          </b-row>
+        </div>
+      </b-container>
+    </section>
+    <section>
+      <b-container class="activity-section">
+        <h2
+          class="activity-section-heading"
+          data-aos="fade-right"
+        >
+          Projects
+        </h2>
+        <div class="activity-cards" data-aos="zoom-in-up">
+          <b-row
+            v-for="index in Math.ceil(projectsData.length / 3) + 1"
+            :key="index"
+          >
+            <b-col sm="12" md="4" v-for="id in 3" :key="id">
+              <Activity
+                class="activity"
+                v-if="getTranslatedId(id, index) < projectsData.length"
+                :title="projectsData[getTranslatedId(id, index)].title"
+                :description="
+                  projectsData[getTranslatedId(id, index)].description
+                "
+                :date="projectsData[getTranslatedId(id, index)].date"
+              />
+            </b-col>
+          </b-row>
+        </div>
+      </b-container>
+    </section>
+  </div>
+</template>
+
+<script>
+import Activity from "../components/activities/Activity";
+import json from "../../database/database.json";
+
+export default {
+  name: "Activities",
+  components: {
+    Activity,
+  },
+  data() {
+    var courses = json.activities.courses;
+    var workshops = json.activities.workshops;
+    var projects = json.activities.projects;
+
+    var i = 0;
```

Comment:
Use `let` or `const` instead of `var`
[Var, Let, and Const – What's the Difference?](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)